docs/src/views/ThemingExamples.tsx

```diff
@@ -277,15 +277,27 @@ export default () => (
       value={`
         <div>
           /* Default theming */
-          <Header as="h3" content="Default" />
-          <Button primary>Branding</Button>
+          <Header as="h3" content="Default theming" />
+          <Button content="Button" />
+          <Button icon="plus" iconOnly primary />
+          <Button icon="at" content="Send email" secondary />
+          <Icon name="chess rook" size="larger" />
+          <Label content="Label with icon" icon="close" />
 
           /* First nested theming */
           <Provider
             theme={{
-              siteVariables: { brand: 'darkred' },
+              componentStyles: {
+                Button: {
+                  root: ({variables, props}) => ({
```

Comment:
Would rather introduce variable for the color value, so that we won't hardcode the `darkred` value. Also, as a pattern and maybe better readability, we can transform the usage into the following:

```
            componentStyles: {
                Button: {
                  root: ({ variables: v, props: p }) => ({
                    ...(p.primary && {
                      background: v.backgroundColor,
                    }),
                  }),
                },
```